From 132f3f7bd214fa16d18e6b92e96279d54bc52d84 Mon Sep 17 00:00:00 2001
From: Steve Loughran <stevel@apache.org>
Date: Thu, 7 Jan 2016 14:51:34 +0000
Subject: [PATCH 2149/2748] HADOOP-12655. TestHttpServer.testBindAddress bind
 port range is wider  than expected. (Wei-Chiu
 Chuang via stevel)

(cherry picked from commit 3190534814587a0e6cb7d99b01cca4480425458b)

Change-Id: Ie256c49d68b61b3ac86257bb293550eb8c68480d
---
 .../org/apache/hadoop/http/TestHttpServer.java     |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/http/TestHttpServer.java b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/http/TestHttpServer.java
index ca0ee6b..67d0caf 100644
--- a/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/http/TestHttpServer.java
+++ b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/http/TestHttpServer.java
@@ -555,9 +555,6 @@ private HttpServer2 checkBindAddress(String host, int port, boolean findPort)
         assertTrue(boundPort != 0); // ephemeral should now return bound port
       } else if (findPort) {
         assertTrue(boundPort > port);
-        // allow a little wiggle room to prevent random test failures if
-        // some consecutive ports are already in use
-        assertTrue(boundPort - port < 8);
       }
     } catch (Exception e) {
       server.stop();
-- 
1.7.9.5

